2008-09-12  Stepan Kasal  <skasal@redhat.com>

	* aclocal.m4 (BRLTTY_HELP_STRING, BRLTTY_TEXT_TABLE,
	BRLTTY_ATTRIBUTES_TABLE): Expand parameters to AC_HELP_STRING
	and m4_text_wrap, to be compatible with Autoconf 2.62+.

--- brltty-3.9.orig/aclocal.m4.orig	2007-10-17 17:19:31.000000000 +0200
+++ brltty-3.9.orig/aclocal.m4	2008-09-12 18:05:21.000000000 +0200
@@ -144,7 +144,7 @@
 
 AC_DEFUN([BRLTTY_HELP_STRING], [dnl
 AC_HELP_STRING([$1], patsubst([$2], [
-.*$]), [brltty_help_prefix])dnl
+.*$]), m4_defn([brltty_help_prefix]))dnl
 patsubst(patsubst([$2], [\`[^
 ]*]), [
 ], [\&brltty_help_prefix])[]dnl
@@ -384,11 +384,11 @@
 
 AC_DEFUN([BRLTTY_TEXT_TABLE], [dnl
 define([brltty_tables_text], ifdef([brltty_tables_text], [brltty_tables_text])[
-m4_text_wrap([$2], [           ], [- m4_format([%-8s ], [$1])], brltty_help_width)])])
+m4_text_wrap([$2], [           ], [- ]m4_format([%-8s ], [$1]), brltty_help_width)])])
 
 AC_DEFUN([BRLTTY_ATTRIBUTES_TABLE], [dnl
 define([brltty_tables_attributes], ifdef([brltty_tables_attributes], [brltty_tables_attributes])[
-m4_text_wrap([$2], [             ], [- m4_format([%-10s ], [$1])], brltty_help_width)])])
+m4_text_wrap([$2], [             ], [- ]m4_format([%-10s ], [$1]), brltty_help_width)])])
 
 AC_DEFUN([BRLTTY_SUMMARY_BEGIN], [dnl
 brltty_summary_lines="Options Summary:"
